crates/cast/tests/e2e/deploy.rs

```diff
@@ -94,7 +94,7 @@ async fn test_contract_not_declared() {
     let snapbox = runner(&args);
     let output = String::from_utf8(snapbox.assert().get_output().stderr.clone()).unwrap();
 
-    assert!(output.contains("Class with hash 0x1 is not declared."));
+    assert!(output.contains("Transaction execution has failed."));
```

Comment:
Isn't it a regression?

Reply:
This is devnet side; I did not check recently, but real nodes did return different messages (and we are not dependent on the message body, only on tx status). Will try to find some time to run cast against integration to check what are the messages, and raise an issue to devnet if there are differences. For now I don't think this is blocking in any way.

Reply:
Doesn't have to be resolved in this PR, but I'd create an issue to check this later.

Reply:
Sure, https://github.com/foundry-rs/starknet-foundry/issues/1000